src/main/java/roomescape/reservation/controller/dto/ReservationWithStatusResponse.java

```diff
@@ -20,29 +21,37 @@ public class ReservationWithStatusResponse {
     private final LocalDate date;
     private final LocalTime time;
     private final String status;
+    private final PaymentHistoryWebResponse paymentHistory;
 
-    public static ReservationWithStatusResponse from(
-            final Reservation reservation
-    ) {
+    public static ReservationWithStatusResponse of(Reservation reservation, PaymentHistory paymentHistory) {
```

Comment:
```suggestion
    public static ReservationWithStatusResponse of(Reservation reservation, @Nullable PaymentHistory paymentHistory) {
```
애노테이션을 통해 nullable한 값임을 명시해주면 더 안전할 것 같아요.

Reply:
적용했습니다!

Reply:
수정했습니다!
6ad750258abb79c8e7af66ec17f40e64c4b726b6